sytest-blacklist

```diff
@@ -29,3 +29,7 @@ Enabling an unknown default rule fails with 404
 
 # Blacklisted due to https://github.com/matrix-org/synapse/issues/1663
 New federated private chats get full presence information (SYN-115)
+
+# Blacklisted due to https://github.com/matrix-org/matrix-doc/pull/2314 removing
+# this requirement from the spec
+Inbound federation of state requires event_id as a mandatory paramater
```

Comment:
probably a better fix is to remove the test from sytest

Reply:
I guess this is debatable until the MSC lands. :man_shrugging:

Reply:
yeah the spec requires event_id as mandatory, after the MSC lands we should def remove from sytest